tests/general/InvalidLinkBearTest.py

```diff
@@ -86,6 +87,24 @@ def assertResultCount(self, test_file, expected_num):
                 outputs = list(uut.run('testline', [line]))
                 self.assertEqual(num, len(outputs))
 
+    def assertSeverity(self, file, severity, settings={}):
+        """
+        Test the warnings in each line of the file to match the
+        given severity.
+        :param file: The ``file`` to be checked.
+        :param severity: The severity level of the warnings in each
+                         line of the file.
+        """
+        severity_tag = RESULT_SEVERITY.reverse[severity]
+        with requests_mock.Mocker() as m:
+            InvalidLinkBear.check_prerequisites = lambda *args: True
```

Comment:
Do you want me to remove complete `assertSeverity` function or just the single line.